sdk/src/Core/Amazon.Runtime/Enumerations.cs

```diff
@@ -48,17 +48,10 @@ public enum ErrorType
     }
 
     /// <summary>
-    /// Sts Regional Endpoints Value determines whether or not
-    /// to send the sts request to the regional endpoint or to
-    /// the global sts endpoint
+    /// StsRegionalEndpointsValue determines the service endpoint used to talk to STS.
     /// </summary>
     public enum StsRegionalEndpointsValue
```

Comment:
Do we even need this enum anymore or can we just delete it. I think we can just delete it because the only reason we had it before was to support the legacy use case. Without the legacy endpoint resolution STS works like any other service and shouldn't need this enum.